dune-project

```
@@ -22,7 +22,7 @@ code required to build a project using the dune build tool.")
   dune-build-info
   base
   bos
-  (cmdliner (< 1.1.0))
+  cmdliner
```

I think we need to have a minimum constraint here:

```suggestion
  (cmdliner (>= 1.1.0))
```